### DISCUSSION TITLE
Mysqldump can't create/write to file

### DISCUSSION THREAD

**Discussion by dylanelliott27** [welaika/wordmove#646] [Q&A]
I am on MacOS using Mamp as my local environment. 

When trying to push my database to staging, I get the error: mysqldump: Can't create/write to file 'path-here' (Errcode: 13 - Permission denied)

I assume this might be because I am using the MySQL from within Mamp, and it doesn't have permission to write to a file outside of Mamp?

Has anyone came across this before or know how to give it permission to write the dump file?

Thank you.

**Comment by alessandro-fazzi** [comment]
```
mysqldump: Can't create/write to file 'path-here' (Errcode: 13 - Permission denied)
```

Seems like your user is not able to write into the `path-here` directory. The output of `ls -la path-here` (I mean: the folder containing the file if `path-here` is the path of a file as I think) wuold be useful both for your and our check.

Keep in mind when you invoke a command, the command itself hasn't permissions: invoking user's permissions apply instead (probably 99% truth).
